packages/stark-ui/src/modules/toast-notification/components/_toast-notification.component.scss

```diff
@@ -0,0 +1,43 @@
+.mat-snack-bar-container {
+  margin: 8px !important;
+  padding: 0 !important;
+}
+
+.stark-toast {
+  min-height: 0;
+  display: flex;
+  white-space: normal;
+  align-items: center;
+  padding: 6px 16px 6px 16px;
+  & span {
+	line-height: 24px;
+	margin: 6px;
+  }
+  & mat-icon {
+	margin: 6px;
+	min-width: 24px;
+	width: 24px;
+	min-height: 24px;
+	height: 24px;
+  }
+  & .stark-toast-action {
+	float: none;
+	margin: 0;
+	padding: 0;
+  }
+  & .stark-toast-text {
+	flex: 1 1 auto;
+  }
+}
+
+.stark-toast-message-error {
```

Comment:
I think we should use the nesting everywhere we can in our stylesheets 😊 
In this way, could you please adapt this and move this style under `.stark-stoast`.
ie: 
```css
.stark-toast {
  // ...
  &.stark-toast-message-error {
    // ...
  }
}
```